Assignment1/UrlReader.java

```diff
@@ -0,0 +1,67 @@
+package Assignment1;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Scanner;
+
+public class UrlReader {
+    public static void urlInput() throws IOException {
+        UrlReader writer = new UrlReader();
+        writer.input();
+    }
+
+    public void input() throws IOException {
+        while (true) {
+            Scanner sc = new Scanner(System.in);
+            BufferedWriter urlwriter = new BufferedWriter(
+                    new FileWriter("D:\\Internship\\P2 - Java Core\\Assignment\\Assignment1\\urls.txt", true));
```

Comment:
If you already place this url.txt file into the assignment then you can use the relative patch

Reply:
> If you already place this url.txt file into the assignment then you can use the relative patch

yes sir I'll correct it.